### PR TITLE
Add Spring Application for Microform V2 using Latest rest-client

### PR DESCRIPTION
**Pull request by skircher**
Upgrades Microform to V2, adds JWT validation, and uses Spring instead of JSP to increase (hopefully) readability.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (skircher) [inline pom.xml:1]: Adding a parent POM for better IDE experience

File: spring-microform/pom.xml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+            <artifactId>spring-boot-starter-actuator</artifactId>
+            <version>3.0.4</version>
+        </dependency>
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
+            <artifactId>spring-boot-starter-thymeleaf</artifactId>
+            <version>2.7.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.projectlombok</groupId>
```
Comment (skircher) [inline spring-microform/pom.xml:38]: Using Lombok to reduce boilerplate on constructors, getters, setters, exception handling to keep the focus on invoking the rest-client, etc.

File: spring-microform/src/main/java/com/cybersource/example/service/JwtProcessorService.java [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+
+@Service
+@RequiredArgsConstructor
+public class JwtProcessorService {
+
+    @Autowired
+    private final MerchantCredentials merchantCredentials;
+
+    @SneakyThrows
+    public String verifyJwtAndGetDecodedBody(final String jwt) {
```
Comment (skircher) [inline spring-microform/src/main/java/com/cybersource/example/service/JwtProcessorService.java:33]: Added step to verify the JWT came from CyberSource

File: spring-microform/src/main/java/com/cybersource/example/service/MicroformService.java [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+    @Autowired
+    private final MerchantCredentials merchantCredentials;
+
+    public String generateCaptureContext(final GenerateCaptureContextRequest captureContextRequest) throws ConfigException, ApiException{
+
+        final ApiClient apiClient = new ApiClient(new MerchantConfig(merchantCredentials.getAsJavaProps()));
+
+        final MicroformIntegrationApi microformApi = new MicroformIntegrationApi(apiClient);
+
+        final String response = microformApi.generateCaptureContext(captureContextRequest);
```
Comment (skircher) [inline spring-microform/src/main/java/com/cybersource/example/service/MicroformService.java:32]: Using the new Microform API

**Review by gnongsie — APPROVED** [review]

**Pull request merged**